test/testsurround.c

```diff
@@ -198,6 +198,31 @@ main(int argc, char *argv[])
 
         SDL_CloseAudioDevice(dev);
     }
+    
+    char *deviceName;
+    SDL_AudioSpec spec;
```

Comment:
please move these to the top for c90 compatibility.

Reply:
i.e.:
```diff
diff --git a/test/testsurround.c b/test/testsurround.c
index 45c9a58..ff55af7 100644
--- a/test/testsurround.c
+++ b/test/testsurround.c
@@ -135,6 +135,8 @@ fill_buffer(void* unused, Uint8* stream, int len)
 int
 main(int argc, char *argv[])
 {
+    char *deviceName;
+    SDL_AudioSpec spec;
     int i;
 
     /* Enable standard application logging */
@@ -198,10 +200,7 @@ main(int argc, char *argv[])
 
         SDL_CloseAudioDevice(dev);
     }
-    
-    char *deviceName;
-    SDL_AudioSpec spec;
-    
+
     if (SDL_GetDefaultAudioInfo(&deviceName, &spec, 0)) {
         SDL_Log("Error when calling SDL_GetDefaultAudioInfo: %s\n", SDL_GetError());
     } else {
```

Reply:
Done.

Reply:
Can't see any new commits here.

Also: if you sqash your commits into a single one and do a force-push,
the git history will be cleaner.

Reply:
Alright, I'm somewhat new to git so I don't know how to squash commits into a single one. How would one do something like this? (In the meantime I'll be pushing the commits which I forgot to do).

Reply:
[This small blog post](https://thoughtbot.com/blog/git-interactive-rebase-squash-amend-rewriting-history#squash-commits-together) explains it all well.
Afterwards you need to force push to your git fork (`git push` with `-f` option).

No need to close this pr and re-open a new one because we can also fix this on our side.

Reply:
Thanks for the info!

Reply:
I squashed your pr to one commit in [52534bb](https://github.com/libsdl-org/SDL/pull/6277/commits/52534bb800bd726564c26fc3308634756a53dadf)